processor/src/main/java/org/mapstruct/ap/internal/util/workarounds/SpecificCompilerWorkarounds.java

```diff
@@ -134,7 +140,8 @@ static TypeMirror asMemberOf(Types typeUtils, ProcessingEnvironment env, Version
             catch ( IllegalArgumentException e ) {
                 lastException = e;
                 if ( versionInformation.isEclipseJDTCompiler() ) {
-                    result = EclipseAsMemberOfWorkaround.asMemberOf( env, containing, element );
+                    result = (TypeMirror) ClassLoaderHelper.getAsMemberOf( element.getClass().getClassLoader() )
```

Comment:
I thought about it, but there are some differences in version counting when comparing the usage within the IDE and when using it via Maven or so. Plus I don't know what version exactly introduced that change. So I felt more safe to do it in a way that works for "all known" versions for now.